docs/conf.py

```diff
@@ -23,8 +24,6 @@
 # version is used.
 sys.path.insert(0, project_root)
 
-import dateparser
-
```

Comment:
From what I read in the comments:

> Insert the project root dir as the first element in the PYTHONPATH.
> This lets us ensure that the source package is imported, and that its
> version is used.

 I think that this import is here intentionally. I'm not sure if moving it from here would have any adversarial consequence. Wouldn't be better to add change it to...
 
  ```python3
  import dateparser  # noqa: E402
  ```

just in case? @Gallaecio

Reply:
BTW, I've seen that in `parsel` has the import above, so maybe I'm worrying unnecessarily:

https://github.com/scrapy/parsel/blob/master/docs/conf.py

Reply:
I had not thought of that. Indeed, having it after will prevent a system-wide-installed version of dateparser being used instead.

It should not be a problem when using `tox` though, I believe, since `tox` installs the project into its virtual environment every time.

Reply:
I prefer to keep it where it was to avoid needing to modify the comments, etc.

@gavishpoddar could you move it again where it was and add the "noqa"?:

`import dateparser  # noqa: E402`

Thanks!

Reply:
Done 👍🏻